src/main/scala/com/hacklanta/formality/FieldHandling.scala

```diff
@@ -238,8 +238,12 @@ abstract class BaseFieldHolder[
     val nameTransform = (selector + " [name]") #> functionId
 
     initialValue.map { startValue =>
-      nameTransform &
-      (selector + " [value]") #> serializeValue(startValue)
+      selector #> { ns: NodeSeq => ns match {
+        case elem: Elem if elem.label == "textarea" =>
+          (nameTransform & "^ *" #> serializeValue(startValue)) apply ns
```

Comment:
Now that I look at this in real life I wonder if we should split it onto two lines:

```scala
(nameTransform &
 "^ *" #> serializeValue(startValue) apply ns
```